### PR TITLE
Moves Milind to team from alumnus

### PR DESCRIPTION
**Pull request by milindmantri**

### CODE REVIEW

**Comment by milindmantri** [comment]
@hari-alvyl, please review. I can't seem to assign any reviewers.

**Review by gvatreya — APPROVED** [review]

**Comment by gvatreya** [comment]
@milindmantri The website still has the (Alumnus) under your name. 
Do you need to do any other changes?

File: content/team-current/milind.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 +++
 
-date = "2018-06-24T00:00:00+05:30" 
+date = "2021-01-12T00:00:00+05:30"
```
Comment (dostiharise) [inline content/team-current/milind.md:3]: @milindmantri 

Along with this, you need to set `is_alumnus = false`.

Refer to line no `9`.

**Comment by milindmantri** [comment]
My bad. I thought I had changed that.
I'll just push the changes.

**Pull request merged**